wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -394,6 +399,41 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:restriction>
 			</xs:simpleType>
 			<!--===============================-->
+			<xs:simpleType name="HardwareTypes">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="Camera">
+						<xs:annotation>
+							<xs:documentation>Single sensor camera. Ex. bullet, PTZ or fisheye camera.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="Multi-sensor">
+						<xs:annotation>
```

Comment:
Would be nice to mention this as **MultiSensorCamera**  because multi-sensor is very generic referring to any kind of sensor.

Reply:
done